LibTest/io/File/openRead_A02_t01.dart

```diff
@@ -30,9 +31,13 @@ _main(Directory sandbox) async {
   File file = getTempFileSync(parent: sandbox);
   file.writeAsBytesSync([1, 2, 3]);
   asyncStart();
-  await file.openRead().listen((data) {
-    Expect.throws(() {file.deleteSync();});
-  }).onDone(() {
+  final c = new Completer();
+  StreamSubscription<List<int>> s = null;
+  s = await file.openRead().listen((data) async {
+    Expect.listEquals([1, 2, 3], data);
```

Comment:
The callback given to `listen((data) {... })` can be called multiple times, each with a subset of the bytes. There is no  guarantee that the very first time we get all 3 bytes, consider changing this to expect that `data` is a prefix of `[1, 2, 3]`